src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -204,5 +204,12 @@
             </property>
             <deprecated>The service "%service_id%" is internal and deprecated since Symfony 3.3 and will be removed in Symfony 4.0</deprecated>
         </service>
+
+        <!-- Commands -->
+        <service id="form.command.debug" class="Symfony\Component\Form\Command\FormDebugCommand">
```

Comment:
I think this command should only be available/registered when debug is true. I wonder though, is there are reason the debug commands are not in the debug bundle and enabled based on the available packages?